dev/io.openliberty.org.jboss.resteasy.mprestclient.4.0/src/org/jboss/resteasy/microprofile/client/ProxyInvocationHandler.java

```diff
@@ -0,0 +1,308 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I'd like to see a comment here with where the class came from (with the correct branch for the version on back level versions)

Since 3.0 is the most recent version, the `main` branch is sufficient.
```suggestion
// https://github.com/resteasy/resteasy-microprofile/blob/main/rest-client-base/src/main/java/org/jboss/resteasy/microprofile/client/ProxyInvocationHandler.java

```

Reply:
I didn't get the file from github.com.  I got it from the source jar from maven that matched the version that is referenced in the bnd.bnd file.

Reply:
So if I were to reference anything I would reference the location in maven instead of github in my opinion.